packages/apollo/src/config.ts

```diff
@@ -82,7 +82,7 @@ function loadSchemaConfig(
       obj.endpoint,
       !obj.engineKey && defaultEndpoint
     ),
-    engineKey: obj.engineKey,
+    engineKey: obj.engineKey || process.env.ENGINE_API_KEY,
```

Comment:
should `env` take precedent?

Reply:
f25428ed1f60a5f66f4f3f906f97052a2047a35f